server/supernova/server/server.cpp

```diff
@@ -269,27 +269,43 @@ static bool set_realtime_priority(int thread_index)
 #ifdef NOVA_TT_PRIORITY_RT
 
 #ifdef JACK_BACKEND
+#ifndef _WIN32
         int priority = instance->realtime_priority();
         if (priority >= 0)
             success = true;
-
+#else
+		int priority = thread_priority_interval_rt().second;
+		success = true;
+#endif
 #elif _WIN32
         int priority = thread_priority_interval_rt().second;
+		success = true;
 #else
         int min, max;
         boost::tie(min, max) = thread_priority_interval_rt();
         int priority = max - 3;
         priority = std::max(min, priority);
 #endif
 
-        if (success)
+        if (success){
+			//std::cout << "setting thread priority " << priority << std::endl;
```

Comment:
please delete unused code rather than commenting it out

Reply:
done